src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,83 @@
+.product-card {
+  box-sizing: border-box;
+  background-color: #fff;
+  width: $card-width;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 0;
+  }
```

Comment:
put padding for  top for your card

Reply:
I have 16px padding in card and add more if it is neccesary. I think it looks better

Reply:
It's a distance between border of the card and it's content, so logically it should be a padding